tools/add_triage_signature.py

```diff
@@ -77,14 +77,20 @@ def days_ago(datestr):
         return 9999
 
 
+class FailedToGetMetadataException(Exception):
+    pass
+
+
 @functools.lru_cache(maxsize=1000)
 def get_metadata_json(cluster_url):
-    res = requests.get("{}/metdata.json".format(cluster_url))
-    res.raise_for_status()
-    return res.json()
+    try:
+        res = requests.get("{}/metdata.json".format(cluster_url))
+        res.raise_for_status()
+        return res.json()
+    except Exception as e:
```

Comment:
maybe only on 404?

Reply:
Already had this discussion in a different PR. I didn't add this `except` clause, I just moved it from the signatures to this function. Actually changing it would cause behavior change that is out of the scope of this PR.